doc/user_guide/docker/docker-usage.md

```diff
@@ -0,0 +1,158 @@
+* On Windows mounting `docker.sock` only works with Docker Desktop with WSL 2.
+```shell
+docker run \
+  --volume ${VOLUME}:/root/notebooks \
+  --volume /var/run/docker.sock:/var/run/docker.sock \
+  --publish ${LISTEN_IP}:49494:49494 \
+  exasol/ai-lab:${VERSION}
+```
+
+## Stopping the AI-Lab Docker Container
+
+If you used one of the commands given in preceding sections without option `--detach` then you can stop the AI-Lab Docker container by simply pressing Ctrl-C.
+
+If you used option `--detach` then you need to use the following command:
+
+```shell
+docker stop <CONTAINER>
+```
+
+See also https://docs.docker.com/engine/reference/commandline/container_stop/.
+
+## Restarting a Stopped Container
+
+In general we recommend to restart a stopped container instead of creating a new one. This has the benefit of keeping [additional dependencies](#installing-additional-dependencies) that you did install:
+
+```shell
+docker start <CONTAINER>
+```
+
+**Please note**
+* If you accidently created a new AI-Lab Docker container and the stopped container still exists please remove the new container and restart the existing one.
+* If there is no stopped AI-Lab container and you are using Exasol Docker-DB you need to link the newly created container to the Exasol Docker-DB network `db_network_DemoDb`.
+
+```shell
+docker network connect db_network_DemoDb <CONTAINER>
+```
+
+See also https://docs.docker.com/engine/reference/commandline/network_connect/.
+
+## Connecting to Jupyter Service
+
+When starting AI-Lab as Docker container the command line will display a welcome message showing connection instructions and a reminder to change the default password:
+
+```
+$ docker run --publish 0.0.0.0:$PORT:49494 exasol/ai-lab:$VERSION
+Server for Jupyter has been started successfully.
+
+You can connect with http://<host>:<port>
+
+If using a Docker daemon on your local machine and did forward the
+port to the same port then you can connect with http://localhost:49494.
+
+┬ ┬┌─┐┌┬┐┌─┐┌┬┐┌─┐  ┬ ┬┌─┐┬ ┬┬─┐   ┬┬ ┬┌─┐┬ ┬┌┬┐┌─┐┬─┐  ┌─┐┌─┐┌─┐┌─┐┬ ┬┌─┐┬─┐┌┬┐ ┬
+│ │├─┘ ││├─┤ │ ├┤   └┬┘│ ││ │├┬┘   ││ │├─┘└┬┘ │ ├┤ ├┬┘  ├─┘├─┤└─┐└─┐││││ │├┬┘ ││ │
+└─┘┴  ─┴┘┴ ┴ ┴ └─┘   ┴ └─┘└─┘┴└─  └┘└─┘┴   ┴  ┴ └─┘┴└─  ┴  ┴ ┴└─┘└─┘└┴┘└─┘┴└──┴┘ o
+
+The default password is "ai-lab".
+To update the password, log in to the Docker container as the user root and run
+    /root/jupyterenv/bin/jupyter-lab server password
+```
+
+Using an internet browser you then can connect to the Jupyter server running in the Docker container in order to follow the tutorials presented by a set of Jupyter notebooks, see [Connecting to Jupyter Service](../jupyter.md#open-jupyter-in-your-browser).
+
+For parameter `<host>`: If your daemon machine is identical to the machine your browser is running on then you can replace `<host>` by `localhost` otherwise please use the IP address of the daemon machine.
```

Comment:
```suggestion
For the parameter `<host>`: If your daemon machine is identical to the machine your browser is running on then you can replace `<host>` by `localhost` otherwise please use the IP address of the daemon machine.
```